Source/DataSources/CzmlDataSource.js

```diff
@@ -217,7 +221,16 @@ define([
     function unwrapUriInterval(czmlInterval, sourceUri) {
         var result = defaultValue(czmlInterval.uri, czmlInterval);
         if (defined(sourceUri)) {
-            result = getAbsoluteUri(result, getAbsoluteUri(sourceUri));
+            var uriComponents = sourceUri.split('?');
```

Comment:
I don't think we want to get the parameters from the sourceUri.  We need to thread the `query` object through all of the calls so that we can use it here instead.